assets/dev/scss/editor/_global.scss

```diff
@@ -166,3 +165,31 @@
 i.eicon-tilted {
 	transform: rotate(90deg);
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
+	margin-top: 15px;
+}
+
+.elementor-control-media-size .elementor-control-field {
+	padding: 0px;
+}
+
+.elementor-control-media-size .elementor-control-responsive-switchers {
+	display: none;
+}
+
+.elementor-control-media-display-size.select {
+	padding-bottom: 0px;
+	width: 72%;
+	float: right;
```

Comment:
float?
can't it be solved with some `display` property?

Reply:
No, I gave it a try first.

Reply:
Are you sure?
There must be a way

Reply:
Worked on the second e try, done.